package.json

```diff
@@ -58,6 +58,7 @@
     "react-copy-to-clipboard": "^4.2.3",
     "react-custom-scrollbars": "^4.0.2",
     "react-dom": "^15.4.2",
+    "react-ionicons": "^1.5.7",
```

Comment:
Is loading own fonts? We have been already using `ionicons` fonts.

Reply:
@ganzorig